### ISSUE TITLE
Containerize an application Tutorial not fully accessible

### ISSUE THREAD

**Issue by scorbitt1** [docker/docs#16192]
File: [get-started/02_our_app.md](https://docs.docker.com/get-started/02_our_app/)

In Containerize Tutorial Left Side Navigation Menu Not Keyboard Accessible.  The dropdown options in the navigation menu are not keyboard accessible.  This violates WCAG Success Criteria item number 2.1.1.  Students who use the keyboard to navigate will have trouble navigating this website

**Comment by scorbitt1** [comment]
Is there a method to get this looked into?

**Comment by aevesdocker** [comment]
Closing this as the docs have changed significantly with efforts to improve A11y. 
If this is still an issue, please open a new issue
